src/per/i2c.h

```diff
@@ -125,8 +143,34 @@ class I2CHandle
                        CallbackFunctionPtr callback,
                        void*               callback_context);
 
+    /** Receives data with a DMA and returns immediately. Use this for larger transmissions.
+     *  The pointer to data must be located in the D2 memory domain by adding the 
+     *  `DMA_BUFFER_MEM_SECTION` attribute like this:
+     *      uint8_t DMA_BUFFER_MEM_SECTION my_buffer[100];
+     *  If that is not possible for some reason, you MUST clear the cachelines spanning the size of 
+     *  the buffer, before initiating the dma transfer by calling 
+     *  `dsy_dma_clear_cache_for_buffer(buffer, size);`
+     * 
+     *  A single DMA is shared across I2C, I2C2 and I2C3. I2C4 has no DMA support (yet).
```

Comment:
`I2C1` instead of `I2C` ?

Reply:
Yeah I saw that and I'm not sure why I didn't correct it. Guess I forgot!